src/dealWizard/wizardManager.ts

```diff
@@ -0,0 +1,116 @@
+import { autoinject } from "aurelia-framework";
+import { PLATFORM } from "aurelia-pal";
+import { RouteConfig } from "aurelia-router";
+import { WizardService, IWizardState, IWizardStage } from "services/WizardService";
+import { DealRegistrationData, IDealRegistrationData } from "entities/DealRegistrationData";
+import { IStageMeta, WizardType, STAGE_ROUTE_PARAMETER } from "./dealWizardTypes";
+
+const proposalStage: IWizardStage = {
+  name: "Proposal",
+  valid: false,
+  route: "proposal",
+  moduleId: PLATFORM.moduleName("./stages/proposalStage/proposalStage"),
+};
+
+const proposalLeadStage: IWizardStage = {
+  name: "Lead Details",
+  valid: false,
+  route: "proposal-lead",
+  moduleId: PLATFORM.moduleName("./openProposalWizard/openProposalProposalLeadStage/openProposalProposalLeadStage"),
+};
+
+const primaryDaoStage: IWizardStage = {
+  name: "Primary DAO",
+  valid: false,
+  route: "primary-dao",
+  moduleId: PLATFORM.moduleName("./stages/primaryDaoStage/primaryDaoStage"),
+};
+
+const partnerDaoStage: IWizardStage = {
+  name: "Partner DAO",
+  valid: false,
+  route: "partner-dao",
+  moduleId: PLATFORM.moduleName("./stages/partnerDaoStage/partnerDaoStage"),
+};
+
+@autoinject
+export class WizardManager {
+  public wizardState: IWizardState<IDealRegistrationData>;
+  public stageMeta: IStageMeta;
```

Comment:
can you please comment this and the other significant properties, as well as providing instructions on how to invoke a wizard?

Reply:
Alright